bogenliga/src/app/modules/shared/components/tables/data-table/unsorted-data-table.component.ts

```diff
@@ -0,0 +1,333 @@
+import {Component, EventEmitter, Input, OnChanges, OnInit, Output, SimpleChanges} from '@angular/core';
+import {TranslatePipe} from '@ngx-translate/core';
+import {isNullOrUndefined} from '@shared/functions';
+import {VersionedDataObject} from '../../../data-provider/models/versioned-data-object.interface';
+import {TruncationPipe} from '../../../pipes';
+import {CommonComponent} from '../../common';
+import {BaseTableSorter} from '../control/base-table-sorter.class';
+import {DefaultTableSorter} from '../control/default-table-sorter.class';
+import {tableConfigWithDefaults} from '../control/table-config-mapper';
+import {TableActionType} from '../types/table-action-type.enum';
+import {TableColumnConfig} from '../types/table-column-config.interface';
+import {TableColumnType} from '../types/table-column-type.enum';
+import {TableConfig} from '../types/table-config.interface';
+import {TableRow} from '../types/table-row.class';
+
+@Component({
+  selector:    'bla-unsorted-data-table',
+  templateUrl: './data-table.component.html',
+  styleUrls:   ['./data-table.component.scss'],
+  providers:   [TranslatePipe, TruncationPipe]
+})
+export class UnsortedDataTableComponent extends CommonComponent implements OnInit, OnChanges {
+  @Input() public config: TableConfig;
+  @Input() public rows: TableRow[] = [];
+  @Input() tableSorter: BaseTableSorter;
```

Comment:
Wofür braucht eine UnsortedDataTableComponent einen tableSorter? Bitte ausbauen! Tabelle soll nicht sortiert werden können.
Stattdessen existiert ein weiterer Bug, dass die Sortierung der Ligatabelle nicht stimmt. D.h. Sortierung in Ligatabelle-Komponente umsetzen (oder eine leere Methode anlegen, die dann derjenige befüllt der den Sortier-Bug behebt) und hier dann nur die vorsortierten Daten rein kippen und anzeigen.
Hier ist so viel Code drin, den's gar nicht braucht weil keine Sortierung :-)